docs/src/documenter_to_vitepress_docs_example.md

```diff
@@ -1,121 +1,118 @@
 # Upgrading docs from Documenter.jl to DocumenterVitepress.jl
 
- **Assuming that your current documentation is based on Documenter.jl, you can upgrade to DocumenterVitepress.jl by following these steps:**
+**Assuming that your current documentation is based on Documenter.jl, you can upgrade to DocumenterVitepress.jl by following these steps:**
 
 Let us suppose that you are working on a package named `Example.jl` that is part of a GitHub organization named `ExampleOrg`.
 
 Then the very first step here is to update the `make.jl` file to follow the DocumenterVitepress.jl format.
 
-1. Go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl :
+1. Go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl:
 
-:::tabs
+   :::tabs
 
-== For Documenter.jl
+   == For Documenter.jl
```

Comment:
Last time I wanna to change this as.

'From Documenter.jl'   ' to DocumenterVitepress jl' in the second tab. Could you do it here please 🙏.

Reply:
I'm confused. Are you suggesting the following change?
```suggestion
   == From Documenter.jl to DocumenterVitePress.jl
```